modules/dynamic-plugins/ref-community-plugins.adoc

```diff
@@ -76,7 +95,7 @@
 |`./dynamic-plugins/dist/backstage-plugin-scaffolder-backend-module-bitbucket-server-dynamic`
 
 
-|Bulk Import  |`https://npmjs.com/package/@janus-idp/backstage-plugin-bulk-import-backend/v/1.3.0[@janus-idp/backstage-plugin-bulk-import-backend]` |1.3.0 
+|Bulk Import  |`https://npmjs.com/package/@janus-idp/backstage-plugin-bulk-import-backend/v/1.4.1[@janus-idp/backstage-plugin-bulk-import-backend]` |1.4.1 
```

Comment:
A suggestion: would it be easier to create attributes so only the attribute needs to be updated, and then wherever that attribute is referenced, whether here or elsewhere in RHDH prod docs, it will always be up-to-date?

Reply:
See response below regarding automatic table generation script which provides the version number.